sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_encode.py

```diff
@@ -650,8 +662,7 @@ def encode_annotations(value):
     return fields
 
 
-def encode_application_properties(value):
-    # type: (Optional[Dict[str, Any]]) -> Dict[str, Any]
+def encode_application_properties(value: Optional[Dict[str, Any]]) -> Dict[str, Any]:
```

Comment:
Application properties have a known value type - so we don't need to use Any. Though keys should probably be `AnyStr`